include/fmt/compile.h

```diff
@@ -536,15 +570,51 @@ template <typename T, typename Char> struct parse_specs_result {
   int next_arg_id;
 };
 
+constexpr int manual_indexing_id() { return -1; }
```

Comment:
Why is it a function?

Reply:
Made it a variable.
I was just confused by all that global variables linkage complexity that remains the same for constexpr variables. But it looks like all compilers would just optimize out this variable anyway.